doc/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
```

Comment:
Directory name: shouldn't this be `docs`, not `doc`?

Reply:
changed now